rocAL_pybind/amd/rocal/fn.py

```diff
@@ -1087,3 +1087,25 @@ def preemphasis_filter(*inputs, border=types.CLAMP, preemph_coeff=0.97, output_d
                     "output_dtype" :output_dtype}
     preemphasis_output = b.PreEmphasisFilter(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
     return (preemphasis_output)
+
+def nonsilent_region(*inputs, rocal_tensor_output_type = types.FLOAT, bytes_per_sample_hint = [0], cutoff_db = -60, reference_power = 0.0, reset_interval = 8192, seed = -1, window_length = 2048):
```

Comment:
rocal_tensor_output_type in the newly introduced functions, change this variable to output_dtype

Reply:
Do we need to specify the type for NSR as it can always be INT/FLOAT only based on RPP kernel